### PR TITLE
New command to show readme

### PR DESCRIPTION
**Pull request by erashish81**

### CODE REVIEW

**Comment by erashish81** [comment]
@aspandey

File: show_readme.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cat README.md
```
Comment (aspandey) [inline show_readme.sh:4]: We can also add help the name of the author of the files.

**Review by aspandey — CHANGES_REQUESTED** [review]
We can also add help the name of the author of the files.

File: show_readme.sh (hunk 1 above)
Comment (aspandey) [reply show_readme.sh:4]: Good Job :+1:

**Review by aspandey — APPROVED** [review]
:+1:

**Pull request merged**